inbox/react/hooks/overview.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: 'Create Custom Inbox UI with React Hooks'
+sidebarTitle: 'Overview'
+---
+
+import UseNotificationsExample from '/snippets/inbox/react/use-notifications-example.mdx';
+import UseUnreadCountExample from '/snippets/inbox/react/use-unread-count-example.mdx';
+import NovuProviderExample from '/snippets/inbox/react/novu-provider-example.mdx';
+
+The `@novu/react` package offers an interface that enables you to build a custom notifications UI using React hooks that are powered by real-time data from the Novu services.
+
+These hooks are designed for use in both mobile and web applications, offering a flexible approach to building a custom notifications UI tailored to your application's requirements.
+
+### Getting Started
+
+Follow these steps to get started with building your custom inbox UI:
+
+<Steps>
+  <Step title="Install the @novu/react package">
+    ```bash 
+    npm install @novu/react 
+    ```
+  </Step>
+  <Step title="Implement the NovuProvider">
+    To implement the [NovuProvider](/inbox/react/hooks/novu-provider) component, you need to place it in your application's code at the tree level where you want the hooks to be accessible. Here's how you can do it:
```

Comment:
Shouldn't we place the `<NovuProvider />` somewhere in the example below?

Reply:
@desiprisg it's there ;) 

![Screenshot 2024-09-18 at 11 22 24](https://github.com/user-attachments/assets/58b91314-1fd3-4f3b-8c0b-d153400ff196)